### ISSUE TITLE
Integrating tags plugin

### ISSUE THREAD

**Issue by JMurphyWeb** [emfoundation/ce100-app#340]
Features that are affected by tags:
Add tags to orgs view
Add tags to chall view
Edit org
Edit challenge
View an organisation (tags on org and challenges)
Explore: (org and chal)
- select a tag to filter by
- filter by tag functionality 
- tag filter at the top

**Comment by JMurphyWeb** [comment]
Some functions that interact with tags:

handlers/helpers.js
helpers.getTagNames
helpers.getChallenge
helpers.getTagFromId

**Comment by JMurphyWeb** [comment]
Some useful functions we can expose on the request objects:

`request.tags.getAllActive()`
`request.tags.getTagFromId(id)`

**Comment by JMurphyWeb** [comment]
helpers.getTagFromId
```js
helpers.getTagFromId = (allTags) => (id) =>
  id && allTags[id[0]] && allTags[id[0]].tags[id[1]] && {
    id: id,
    name: allTags[id[0]].tags[id[1]].name
  };

```

**Comment by SimonLab** [comment]
resolved with #440
